### DISCUSSION TITLE
开发库命名规范

### DISCUSSION THREAD

**Discussion by Iceyer** [linuxdeepin/developer-center#3296] [Developer Discussion | 开发者讨论]
由于目前项目比较多，需要考虑对应用/系统组件/开发库（dtk/qt 非dtk）的项目命名进行规范，参考如下：

| 分类 | 项目名称 | Git仓库名称 | 包名 | so名称 |
|--|--|--|--|--|
|DTK项目| DtkCore | dtkcore | libdtkcore5 | libdtkcore.so.5.5.29|
| 待补充 |

 #31

**Comment by zccrs** [comment]
项目名会在哪里体现？文档类的地方吗

**Reply by Iceyer** [reply]
构建出来的文件、包名、文档应该都包含。需要给一些实际的例子来指导子项目命名。

**Comment by BLumia** [comment]
有一个相关的 issue： https://github.com/linuxdeepin/developer-center/issues/3540
